src/routes/safe/components/Transactions/TxList/hooks/useGetTxTo.ts

```diff
@@ -0,0 +1,18 @@
+import { Transaction } from 'src/logic/safe/store/models/types/gateway.d'
+
+export const useGetTxTo = (tx: Transaction): string | undefined => {
```

Comment:
why does it use `use*` naming? it looks like a regular function to me

Reply:
thx, good catch!